core/tools.js

```diff
@@ -1518,6 +1518,71 @@
 		},
 
 		/**
+		 * Converts hexstring to base64 coode.
+		 *
+		 * Hexstring contains 8bit numbers, where base64 use 6bit to code.
+		 * We need to process 3 * 8bit chunks of hexstring into 4 * 6bit characters of base64.
+		 * Algorithm:
+		 * 1. Take 3 * 8bit.
+		 * 2. If there is less than 3 byte, fill it with zeros.
+		 * 3. Transform 3 * 8bit into 4 * 6bit numbers.
+		 * 4. Translate those numbers to proper character related to base64.
+		 * 5. If there was added extra zeros bytes fill them with '=' sign.
+		 *
+		 * Example:
+		 * 	hex: 08A11D8ADA2B -> binary: 0000 1000 1010 0001 0001 1101 1000 1010 1101 1010 0010 1011
+		 * 	| <- shows where base64 will cut bits
+		 * 	binary: 0000 10|00 1010| 0001 00|01 1101| 1000 10|10 1101| 1010 00|10 1011
+		 * 	This gives indexes in base64character table
+		 * 	decimal: 2 10 4 29 34 45 40 43 -> base64: "CKEditor"
+		 *
+		 * @since 4.8.0
+		 * @param {String} hexstring string contained only hexadecimal values, e.g. "0AB7"
+		 * @returns {String} base64 string
+		 */
+		hexstring2base64: function( hexstring ) {
```

Comment:
When picking a name, check the surrounding names. Any other convert methods uses `convertXyzToXyz` so you're missing convert prefix and it should be "to" rather than "2".

Reply:
Use consistent letter casing: `hexString` rather than `hexstring`.